atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/OneNodeDownTestSuite.java

```diff
@@ -33,11 +36,16 @@
         OneNodeDownTableManipulationTest.class,
         OneNodeDownNodeAvailabilityTest.class
         })
+@ShouldRetry(numAttempts = 3)
 public final class OneNodeDownTestSuite extends NodesDownTestSetup {
 
+    // Reusing containers when the FlakeRule kicks in.
     @ClassRule
-    public static final Containers CONTAINERS = new Containers(NodesDownTestSetup.class)
-            .with(new ThreeNodeCassandraCluster());
+    public static final RuleChain ruleChain = RuleChain
+            .outerRule(new Containers(NodesDownTestSetup.class)
+                    .with(new ThreeNodeCassandraCluster()))
+            .around(SchemaMutationLockReleasingRule.createChainedReleaseAndRetry(
+                    createCassandraKvs(), CONFIG));
```

Comment:
ooh, I didn't know you can wrap a RuleChain in a RuleChain (though it kind of makes sense!)

Reply:
:yodawg: